app/routes.js

```diff
@@ -26,13 +26,13 @@ const secEvents = require('./middleware/secEvents');
 module.exports = (app, passport, mongoose) => {
 
     // setup resume repo password for Companies Only
-    const basic = auth.basic({
-        realm: 'Sponsoring companies only.',
-        file: __dirname + '/users.htpasswd'
-      });
+    // const basic = auth.basic({
```

Comment:
Normally when using git or some other version control system, you don't comment out code to "remove" it since it's tracked in git.